.github/scripts/execute_single_benchmark.sh

```diff
@@ -111,7 +112,7 @@ elif [ "$DATASET_NAME" == "skydio-501" ]; then
   tar -xvzf skydio-crane-mast-501-images1.tar.gz
   tar -xvzf skydio-crane-mast-501-images2.tar.gz
   tar -xvzf skydio-501-colmap-pseudo-gt.tar.gz
-  IMAGES_DIR="skydio-crane-mast-501-images"
+  IMAGES_DIR="skydio-crane-mast-501-imagevesta_99846.plys"
```

Comment:
Thanks for the PR. Should this go under the astronet section in line 140, instead of the skydio-501 section?